lms/templates/widgets/cookie-consent.html

```diff
@@ -0,0 +1,28 @@
+<!-- Cookie Consent -->
+<%page expression_filter="h"/>
+<%! from django.utils.translation import ugettext as _ %>
+<%namespace name='static' file='../static_content.html'/>
+ 
+<link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.1/cookieconsent.min.css" />
+<script src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.1/cookieconsent.min.js"></script>
+
+<script>
+  var message = "${_('This website uses cookies to ensure you get the best experience on our website.')}";
+  var learn = "${_('Learn more')}";
+  var gotit = "${_('Got it!')}"
+
+  window.cookieconsent.initialise({
+    window: '<div dir="${static.dir_rtl()}" role="dialog" aria-live="polite" aria-label="cookieconsent" aria-describedby="cookieconsent:desc" class="cc-window {{classes}}"><!--googleoff: all-->{{children}}<!--googleon: all--></div>',
+
+    palette:{
+      popup: {background: "#323538", text: "#ffffff"},
+      button: {background: "#007db8", text: "#ffffff"},
```

Comment:
@cahrens, I have tested the colour contrast between the background and text colours for both the button and the popup using this [site](http://webaim.org/resources/contrastchecker/). I hope this is now in order?